pocket_coffea/parameters/jet_scale_factors.yaml

```diff
@@ -33,3 +33,23 @@ jet_scale_factors:
         file: /cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/JME/2018_UL/jmar.json.gz
         name: PUJetID_eff
 
+  jet_puId_WPs:
+      # Jet PU ID working points for different periods, due to bug values are different for 2016 
+      # (see here https://twiki.cern.ch/twiki/bin/view/CMS/PileupJetIDUL#Trainings)
```

Comment:
Looking at it we should do something like : 

```
jet_puId:
       2016_PreVFP:
           file:  xxxxxxxxxxxxxxxxxxxxxxxxx
           working_point: 
                  L : 1
                  M: 3
                  T: 7
```

What do you think? The function should be changed accordingly but should be straight forward.